src/mesh/FloodingRouter.cpp

```diff
@@ -17,7 +17,7 @@ ErrorCode FloodingRouter::send(MeshPacket *p)
     return Router::send(p);
 }
 
-bool FloodingRouter::shouldFilterReceived(MeshPacket *p)
+bool FloodingRouter::shouldFilterReceived(const MeshPacket *p)
```

Comment:
'const' was removed in #924, but this can now be reverted, since I removed ACK-ing repeated incoming messages.